2/contrib/jenkins/kube-slave-common.sh

```diff
@@ -142,6 +146,98 @@ function generate_kubernetes_config() {
           <imagePullSecrets/>
           <nodeProperties/>
         </org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
+        <org.csanchez.jenkins.plugins.kubernetes.PodTemplate>
+          <inheritFrom></inheritFrom>
+          <name>java-sidecar</name>
```

Comment:
`maven-agent` or other propositions could be `maven-node` or `java-node`

Reply:
From office hours - `java-builder`